<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EyeTracker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/eye.png</normaloff>:/icons/icons/eye.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="playbackGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Playback</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Play</string>
     </property>
     <property name="statusTip">
      <string>Plays the selected session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="heatMapButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Heat map</string>
     </property>
     <property name="statusTip">
      <string>Displays heat map for the selected session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Heat map</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/heatmap.png</normaloff>:/icons/icons/heatmap.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Delete</string>
     </property>
     <property name="statusTip">
      <string>Deletes the selected session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="recordGroup">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Record</string>
    </property>
    <widget class="QLabel" name="videoPlaceholder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200)</string>
     </property>
     <property name="text">
      <string>Press Start</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Start (S)</string>
     </property>
     <property name="statusTip">
      <string>Starts/stops the video feed from camera...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/webcam.png</normaloff>:/icons/icons/webcam.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>S</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrateButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Calibrate (C)</string>
     </property>
     <property name="statusTip">
      <string>Starts calibration process...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/calibrate.png</normaloff>:/icons/icons/calibrate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>C</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Record (R)</string>
     </property>
     <property name="statusTip">
      <string>Record a new session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/record.png</normaloff>:/icons/icons/record.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>R</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="videoCanvas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="displayGroup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="equalizedButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>7</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show equalized image (1)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Equalized</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="thresholdedButton">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>7</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show thresholded image (2)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Thresholded</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="blobsButton">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>7</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show BLOBs (3)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>BLOBs</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="pupilButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>7</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show detected pupil (4)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Pupil</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/about.png</normaloff>:/icons/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>startToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrateButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>calibrateToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>recordToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heatMapButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>heatMapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>equalizedButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>equalizedClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdedButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>thresholdedClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pupilButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pupilClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blobsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>blobsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>listItemSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutClicked()</slot>
  <slot>exitClicked()</slot>
  <slot>startToggled(bool)</slot>
  <slot>calibrateToggled(bool)</slot>
  <slot>recordToggled(bool)</slot>
  <slot>heatMapClicked()</slot>
  <slot>equalizedClicked()</slot>
  <slot>thresholdedClicked()</slot>
  <slot>blobsClicked()</slot>
  <slot>pupilClicked()</slot>
  <slot>playClicked()</slot>
  <slot>listItemSelected()</slot>
  <slot>deleteClicked()</slot>
 </slots>
</ui>
